product-service/coverage/clover.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think there's no point to commit the "coverage" folder since it's autogenerated. If you still need it for some reason it's better to commit it in a separate commit.